wfl/DBManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using wfl.Models;
+
+namespace wfl
+{
+    public class DBManager
```

Comment:
I could look into that. I think the only place where I'm specifying which database I'm using is in the connection string in web.config. I'm wondering if the code would somehow have to be able to determine which DBManager class to use at runtime somehow using that connection string, unless there's some other way to determine which database provider we are accessing at runtime.